app/src/main/java/com/github/se/signify/model/user/UserRepositoryFireStore.kt

```diff
@@ -373,4 +373,48 @@ class UserRepositoryFireStore(private val db: FirebaseFirestore) : UserRepositor
         .addOnSuccessListener { onSuccess() }
         .addOnFailureListener { onFailure(it) }
   }
+
```

Comment:
this function keeps tracking the quest for the initial date when we started to do them.
Code clean with some explanation. Good job